src/components/pages/Venue/VenueEdition/VenueProvidersManager/VenueProvidersManager.jsx

```diff
@@ -46,37 +36,20 @@ const VenueProvidersManagerContainer = ({ notify, venue }) => {
       const selectedProviderId = event.target.value
       const selectedProvider = providers.find(provider => provider.id === selectedProviderId)
       setIsAllocineProviderSelected(false)
-      setProviderSelectedIsFnac(false)
-      setProviderSelectedIsLibraires(false)
-      setProviderSelectedIsPraxiel(false)
-      setProviderSelectedIsTitelive(false)
+      setVenueIdAtOfferProviderIsRequired(selectedProvider?.requireProviderIdentifier)
```

Comment:
```suggestion
      setProviderIdentifierIsRequired(selectedProvider?.requireProviderIdentifier)
```
pour alléger un peu le tout ?